components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/js/operator-finder/operator-finder.js

```diff
@@ -0,0 +1,443 @@
+/**
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+define(['jquery', 'lodash', 'log', 'handlebar', 'designViewUtils'],
+    function ($, _, log, Handlebars, DesignViewUtils) {
+
+        /**
+         * Loads operators from the endpoint.
+         *
+         * @param onSuccess Callback for success handler
+         * @param onError Callback for error handler
+         */
+        var loadOperators = function (onSuccess, onError) {
+            var operators = [];
+            var namespaces = [];
+            $.ajax({
+                type: 'GET',
+                url: window.location.protocol + '//' + window.location.host + '/editor/metadata',
```

Comment:
there is a already a place we get the extensions available . Shall we use that rather calling again to get the meta data ?